framework/assets/yii.gridView.js

```diff
@@ -237,7 +237,8 @@
      * @param {string} selector jQuery selector
      * @param {function} callback The actual function to be executed with this event
      */
-    function initEventHandler($gridView, type, event, selector, callback) {
+    function initEventHandler($gridView, type, event, selector, callback)
```

Comment:
It's OK to be validated but the standard should not be the same as for PHP. Common placement of `{` in JavaScript is same line.